src/deprecated.jl

```diff
@@ -0,0 +1,24 @@
+# Device module
+module Device
+using ClimaComms
+using CUDA
+
+Base.@deprecate device(; disablegpu = false) ClimaComms.device(;
+    disablegpu = false,
+) false
```

Comment:
ClimaComms.device doesn't take a `disablegpu` arg.
```suggestion
Base.@deprecate device(; disablegpu = false) if disablegpu
    ClimaComms.CPUDevice()
else
    ClimaComms.device()
end false
```

Reply:
Ah, whoops, I thought I had looked at the definition in ClimaComms, must have been in ClimaCore. Thanks!